test/lib/ufe/testContextOps.py

```diff
@@ -362,8 +362,13 @@ def testAddNewPrimWithDelete(self):
         cmds.pickWalk(d='down')
         cmds.delete()
 
-        # The proxy shape should now have no UFE child items (since we skip inactive).
-        self.assertFalse(proxyShapehier.hasChildren())
+        # The proxy shape should now have no UFE child items (since we skip inactive)
+        # but hasChildren still reports true for inactive to allow the caller to then
+        # do conditional inactive filtering.
+        if mayaUtils.mayaMajorVersion() >= 2023:
```

Comment:
I'm confused why this check is against a Maya version. What change is there in Maya that is affecting this?

Reply:
I'm confused as to why we changed the test at all.

Reply:
The behaviour of the delete command vs prim changed from deactibvating to really deleting. When deactivated, hasChildren() returns true.

Reply:
Okay then instead of using then the Maya test should also surround lines 363 (which does the delete) and the comment you wrote here should be added to the test. Then for Maya 2023 we should not use delete, but instead deactivate the prim.